app/javascript/components/SearchQueryDisplay.js

```diff
@@ -36,19 +39,36 @@ function formatFacet(facet, index, numFacets) {
     </span>
   )
 }
+export const ClearAllButton = () => {
+  const selectionContext = useContext(SearchSelectionContext)
+  const searchContext = useContext(StudySearchContext)
 
-export default function SearchQueryDisplay({terms, facets}) {
+  const clearSearch = () => {
+    const defaultSearchParams = {
+      terms: '',
+      facets: {}
+    }
+    selectionContext.performSearch()
```

Comment:
this first `performSearch()` is unnecessary -- it's just executing a search with the previously selected params, which is a no-op